core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -350,22 +354,32 @@ public JobDetail executeJob(final Job job, final Context context) throws Operati
         return new GraphResult<>(result, clonedContext);
     }
 
-    private void updateOperationChainView(final Operations<?> operations) {
+    void updateOperationChainView(final Operations<?> operations) {
```

Comment:
Is this change needed? If it's just for the test then it would be better to indirectly call it from one of the public methods

Reply:
Sure thing, this is what I was trying to get at in the PR description - I'll refactor the test to work on the execute method and revert the visibility of the method.